api/src/Entity/ContentNode/SingleText.php

```diff
@@ -35,20 +35,25 @@
     normalizationContext: ['groups' => ['read']],
 )]
 #[ORM\Entity(repositoryClass: SingleTextRepository::class)]
-#[ORM\Table(name: 'content_node_singletext')]
 class SingleText extends ContentNode {
-    #[InputFilter\CleanHTML]
-    #[Groups(['read', 'write'])]
-    #[ORM\Column(type: 'text', nullable: true)]
-    public ?string $text = null;
+    public const JSON_SCHEMA = [
+        'type' => 'object',
+        'additionalProperties' => false,
+        'required' => ['text'],
+        'properties' => [
+            'text' => [
+                'type' => 'string',
+            ],
+        ],
+    ];
 
     /**
-     * @param SingleText $prototype
-     * @param EntityMap  $entityMap
+     * Holds the actual data of the content node
+     * (overridden from abstract class in order to add specific validation).
      */
-    public function copyFromPrototype($prototype, $entityMap): void {
-        parent::copyFromPrototype($prototype, $entityMap);
-
-        $this->text = $prototype->text;
-    }
+    #[ApiProperty(example: ['text' => 'dummy text'])]
```

Comment:
```suggestion
    #[ApiProperty(example: ['text' => 'my example text'])]
```
Das kommt in die OpenAPI Dokumentation, daher sollte es neuen Entwickler*innen helfen zu verstehen dass das ein Freitext-Teil des Payloads ist.

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/2825/commits/dd67ee301305c9596c196ba0562d973d13fb0c69